packages/hoppscotch-common/src/helpers/teams/TeamCollectionAdapter.ts

```diff
@@ -1089,7 +1094,10 @@ export default class NewTeamCollectionAdapter {
       const parentFolderAuth = data.auth
       const parentFolderHeaders = data.headers
 
-      if (parentFolderAuth?.authType === "inherit" && path.length === 1) {
+      if (
+        parentFolderAuth?.authType === "inherit" &&
+        [...path.slice(0, i + 1)].length === 1
```

Comment:
Isn't it redundant to create an array since `path.slice` returns the array already?